### DISCUSSION TITLE
Anyway to enable internal links?

### DISCUSSION THREAD

**Discussion by sodiumchl** [mike-edel/ID-MultiPageImporter#36] [Q&A]
I'm glad that I found this script. Thank you!

I would like to use it to convert a bunch of PDF's into fixed-format EPUBs. The PDF has a TOC and internal crossref links. Is there anyway to import/enable these in the InDesign file? Or is this the wrong way to do this?

**Comment by mike-edel** [comment]
99% sure that's not possible. 
The script just automates the regular "place" feature of InDesign and to my knowledge placing PDF files does not support any interactive features in the placed document whatsoever.

You may be able to achieve this by adding a layer on top with some invisible frames + adding hyperlinks to them. At least that's the way I used to add such features to interactive PDFs.

**Comment by sodiumchl** [comment]
Thanks, Mike. Looks like this is a good package for my purpose
https://github.com/dodeeric/pdf2epubEX
